Description: Adjust to id calculation issues
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 planet-venus (0~git9de2109-1) UNRELEASED; urgency=low
 .
   * New upstream version.
Author: Olivier Berger <obergix@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- planet-venus-0~git9de2109.orig/tests/data/reconstitute/enclosure.xml
+++ planet-venus-0~git9de2109/tests/data/reconstitute/enclosure.xml
@@ -1,6 +1,6 @@
 <!--
 Description:  enclosure
-Expect:       links[0].rel == 'enclosure' and id == 'http://example.com/1'
+Expect:       links[0].rel == 'enclosure' and links[0].href == 'http://example.com/1'
 -->
 
 <rss>
--- planet-venus-0~git9de2109.orig/tests/data/reconstitute/gr_id.xml
+++ planet-venus-0~git9de2109/tests/data/reconstitute/gr_id.xml
@@ -1,6 +1,6 @@
 <!--
 Description:  id
-Expect:       id == 'http://example.com/2'
+Expect:       id == 'http://example.com/1'
 -->
 
 <feed xmlns="http://www.w3.org/2005/Atom">
